InnerEye/ML/pipelines/scalar_inference.py

```diff
@@ -53,7 +54,7 @@ class ScalarInferencePipeline(ScalarInferencePipelineBase):
     """
 
     def __init__(self,
-                 model: BaseModelOrDataParallelModel,
+                 model: BaseModelOrDataParallelModelOrDeviceAwareModule,
```

Comment:
can we use a shorter alias here please? also all of these inherit from DeviceAware, so why not just use that?

Reply:
Done.